### ISSUE TITLE
RDP wrap Win10 10.0.18362.657

### ISSUE THREAD

**Issue by Mark955** [stascorp/rdpwrap#1008]
My RDP wrap on win 10 doesnt work. I tried everythink. Can you help me please? 

![SCR1](https://user-images.githubusercontent.com/61413782/75148359-ba45b080-56ff-11ea-943e-2a91089374d2.png)

![SCR2](https://user-images.githubusercontent.com/61413782/75148369-be71ce00-56ff-11ea-9dd7-818343c7836d.png)

**Comment by Smartaccounts** [comment]
You may need to update rdpwrap.ini under the path C:\Program Files\RDP Wrapper. 

Does you rdpwrap.ini have [10.0.18362.657] and [10.0.18362.657-SLInit]? I had same issue before. I am not developer i think those lines do something for this version of windows 10.

**Comment by monta990** [comment]
Use the most recent rdpwrap.ini
In every new windows version, please take a look of it.

https://raw.githubusercontent.com/asmtron/rdpwrap/master/res/rdpwrap.ini

**Comment by Dev3389** [comment]
RDP Does not work on Win 10 Home. Finally got Comcast Gateway to keep port 3389 open. I can remote locally using 127.0.0.1:3389. Port checks all show 3389 is open. But when i try to access from outside my network it wont connect using RDP.

![image](https://user-images.githubusercontent.com/61475851/75283030-023c0480-57e0-11ea-9df2-74a9fa2f49b9.png)

I have checked Group Policy and deleted any users. I have a local user account as Admin. The only thing i can see is that the Remote Desktop section is missing only shows Allow Remote Assistance.

So basically port 3389 is open, i have looked all over the site and web to allow what i can find. The rdpwrap.ini is current, i have tried several outside computers from friends and family to RDP but nothing will connect. Any ideas? It would be greatly appreciated. I have used Pro for a long time but recently moved and and reloaded Win 10 Home same version in this thread and cant afford another version of Pro. Its insane MS took this basic feature out of Win 10.

Kind Regards,

Dev3389

**Comment by monta990** [comment]
@Dev3389 Dev3389 Win 10 Home not support RDP incomming connections.

**Comment by monta990** [comment]
"Beginning with the Creators Update for Windows 10 Home, RDP Wrapper will no longer work, claiming that the listener is [not listening] because of rfxvmt.dll is missing" - stascorp,  check the main page in te faq section.

https://github.com/asmtron/rdpwrap

**Comment by ormanya** [comment]
Doesn't work for me either ( 10.0.18362.657). When trying to connect via RDP while another user is connected via console, I have to force the console user to disconnect in order to make the RDP connection.

- I am using Windows 10 Pro https://puu.sh/FeRyK/c33c0d5989.png
- I am using the latest rdpwrap.ini linked above.
- RDPConf.exe indicates that the wrapper is installed and functioning https://puu.sh/Ff5Yo/5c3649420a.png
- I have tried rebooting
- I have even tried the Group Policy edits recommended elsewhere https://puu.sh/Ff5ZL/af19f46e55.png

Also tried uninstalling KB4532693 on the host machine

**Comment by 700nader007** [comment]
https://github.com/stascorp/rdpwrap/issues/988#issuecomment-597151413

**Comment by sebaxakerhtc** [comment]
Duplicate of https://github.com/stascorp/rdpwrap/issues/988
